lib/netty

```diff
@@ -0,0 +1 @@
+Subproject commit 871ce43b1f431536e08a36d420875494ef3e9451
```

Comment:
Remove this; we got rid of the submodule and this is residue left over in your checkout.